pubspec.yaml

```diff
@@ -11,13 +11,13 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  package_info_plus: ^1.0.0
+  package_info_plus: ^1.4.0
```

Comment:
Do we need `^1.4.0` for the changes? If not, then probably better to leave it unchanged for better compatibility.

Reply:
Why? There is no compatibility change between 1.0.0 and 1.4.0. They both support the same Flutter and Dart SDK. The only difference is that 1.4.0 removes some deprecated functions that this plugin doesn't use anyway so it would be good to use the latest version. Using 1.0.0 will also give warnings when building about an outdated Android embedding being used.

Reply:
With `^1.0.0`, the version resolver will end up using the latest version anyway assuming there's no version conflicts with other packages. 

In my opinion, this is more flexible for apps/other packages using this package. Imagine if the other app/package depends on version `1.3.0` for some reason. Then there will be version conflicts when they try to install both packages.

But you mentioned that `1.0.0` was giving some warnings so I guess it's good to update it now. Would you be able to update the changelog to include:
```
chore: Updated dependency `package_info_plus: ^1.4.0`
```

Reply:
I understand, and i think it's a matter of personal choice to decide between more compatibility or being stricter but using newer / updated versions of library. I've updated the changelog !